### ISSUE TITLE
Non ASCII characters displayed wrong, if coming from variable

### ISSUE THREAD

**Issue by shameister** [Microsoft/reactxp#722]
Hey people,

I ran into a problem, when displaying content from a variable, which contains non ASCII characters like umlauts. 

If I write my string containing umlauts directly into render everything works fine, but if it comes from a variable the problems start. I reduced my problem to this working example.

**Output**
> <img width="313" alt="bildschirmfoto 2018-06-26 um 13 49 13" src="https://user-images.githubusercontent.com/98413/41909683-f3ae7d58-7947-11e8-9d2c-df5357fbcc85.png">

**Code**
> ```
> let exampleText = 'text from var äöü'
> return (
>     <RX.Text>static text äöü - { exampleText }</RX.Text>
> )
> ```

My file is saved as UTF-8. I am running the web-based output in an electron.

Looking forward to a workaround or solution.

Best,
Sascha

**Comment by erictraut** [comment]
I'm not able to repro the problem. Which platform?

In practice, I never combine text literals with text variables the way you have in your code because it results in multiple elements, and the way in which the JSX compiler strips spaces isn't clear. Instead, I'd write it as follows:

```
<RX.Text>{ 'static text äöü - ' + exampleText }</RX.Text>
```

See if that fixes it.

**Comment by shameister** [comment]
Hey Eric, I am on a Mac (10.13.5). In my original code, which has also the problem the texts are not combined. I just did it to explain the issue. I'd like to decide, which errorText to show to the user dynamically. Like here:


![bildschirmfoto 2018-06-26 um 16 42 54](https://user-images.githubusercontent.com/98413/41920011-1400de08-7960-11e8-8923-fdfad38012fd.png)

**Comment by erictraut** [comment]
What is your target platform? Is it web? Or iOS?

**Comment by shameister** [comment]
The platform I experienced this is **Web inside an electron**.

**Comment by erictraut** [comment]
Can you repro the problem in a browser? I'm not able to.

**Comment by shameister** [comment]
Hey @erictraut ,

sorry for running late. Holiday was waiting. Unfortunately I run into the same issue, when running the page directly in a web browser (FF, Chrome, Safari).

Found out, that if adding the charset-meta tag to the index.html's header, than everything works properly: `<meta charset='utf-8'>`

I started with code from the [samples-section](https://github.com/Microsoft/reactxp/tree/master/samples). Maybe this line can be added?

Regards,
Sascha
